spec/ci-spec.js

```diff
@@ -12,133 +6,115 @@ const express = require('express');
 const wrench = require('wrench');
 const CSON = require('season');
 const apm = require('../lib/apm-cli');
-const {
-  nodeVersion
-} = require('./config.json');
+const { nodeVersion } = JSON.parse(fs.readFileSync('./config.json'));
 
-describe('apm ci', function() {
+describe('apm ci', () => {
   let [atomHome, resourcePath, server] = Array.from([]);
 
-  beforeEach(function() {
+  beforeEach(() => {
     spyOnToken();
     silenceOutput();
-
     atomHome = temp.mkdirSync('apm-home-dir-');
     process.env.ATOM_HOME = atomHome;
 
     resourcePath = temp.mkdirSync('atom-resource-path-');
     process.env.ATOM_RESOURCE_PATH = resourcePath;
-
     delete process.env.npm_config_cache;
-
     const app = express();
-    app.get(`/node/${nodeVersion}/node-${nodeVersion}-headers.tar.gz`, (request, response) => response.sendFile(path.join(__dirname, 'fixtures', 'node-dist', `node-${nodeVersion}-headers.tar.gz`)));
-    app.get(`/node/${nodeVersion}/win-x86/node.lib`, (request, response) => response.sendFile(path.join(__dirname, 'fixtures', 'node-dist', 'node.lib')));
-    app.get(`/node/${nodeVersion}/win-x64/node.lib`, (request, response) => response.sendFile(path.join(__dirname, 'fixtures', 'node-dist', 'node_x64.lib')));
-    app.get(`/node/${nodeVersion}/SHASUMS256.txt`, (request, response) => response.sendFile(path.join(__dirname, 'fixtures', 'node-dist', 'SHASUMS256.txt')));
-    app.get('/test-module-with-dependencies', (request, response) => response.sendFile(path.join(__dirname, 'fixtures', 'install-locked-version.json')));
-    app.get('/test-module', (request, response) => response.sendFile(path.join(__dirname, 'fixtures', 'install-test-module.json')));
-    app.get('/native-module', (request, response) => response.sendFile(path.join(__dirname, 'fixtures', 'native-module.json')));
-    app.get('/tarball/test-module-with-dependencies-1.1.0.tgz', (request, response) => response.sendFile(path.join(__dirname, 'fixtures', 'test-module-with-dependencies-1.1.0.tgz')));
-    app.get('/tarball/test-module-1.1.0.tgz', (request, response) => response.sendFile(path.join(__dirname, 'fixtures', 'test-module-1.1.0.tgz')));
-    app.get('/tarball/native-module-1.0.0.tgz', (request, response) => response.sendFile(path.join(__dirname, 'fixtures', 'native-module-1.0.0.tgz')));
-
+    app.get(`/node/${nodeVersion}/node-${nodeVersion}-headers.tar.gz`, (_request, response) => response.sendFile(path.join(__dirname, 'fixtures', 'node-dist', `node-${nodeVersion}-headers.tar.gz`)));
```

Comment:
Is there a reason that we are using `_request` here rather than `request` like in the original decaf?

Reply:
It indicates that the variable is unused. This is something that the default VSC linter respects, for example.

Reply:
Fair enough, that sounds awesome to me. Just double checking